docs/variables.md

```diff
@@ -85,46 +85,46 @@ For instructions on using variable substitutions see the relevant section of [th
-| `Task`        | `spec.volumes[].projected.sources.serviceaccounttoken.audience` |
-| `Task`        | `spec.volumes[].csi.nodepublishsecretref.name`                  |
-| `Task`        | `spec.volumes[].csi.volumeattributes.* `                        |
+| `Task`        | `spec.volumes[].projected.sources.serviceAccountToken.audience` |
+| `Task`        | `spec.volumes[].csi.nodePublishSecretRef.name`                  |
+| `Task`        | `spec.volumes[].csi.volumeAttributes.* `                        |
 | `Task`        | `spec.sidecars[].name`                                          |
 | `Task`        | `spec.sidecars[].image`                                         |
 | `Task`        | `spec.sidecars[].imagePullPolicy`                               |
 | `Task`        | `spec.sidecars[].env.value`                                     |
-| `Task`        | `spec.sidecars[].env.valuefrom.secretkeyref.name`               |
-| `Task`        | `spec.sidecars[].env.valuefrom.secretkeyref.key`                |
-| `Task`        | `spec.sidecars[].env.valuefrom.configmapkeyref.name`            |
-| `Task`        | `spec.sidecars[].env.valuefrom.configmapkeyref.key`             |
-| `Task`        | `spec.sidecars[].volumemounts.name`                             |
-| `Task`        | `spec.sidecars[].volumemounts.mountpath`                        |
-| `Task`        | `spec.sidecars[].volumemounts.subpath`                          |
+| `Task`        | `spec.sidecars[].env.valueFrom.secretKeyRef.name`               |
+| `Task`        | `spec.sidecars[].env.valueFrom.secretKeyRef.key`                |
+| `Task`        | `spec.sidecars[].env.valueFrom.configMapKeyRef.name`            |
+| `Task`        | `spec.sidecars[].env.valueFrom.configMapKeyRef.key`             |
+| `Task`        | `spec.sidecars[].volumeMounts.name`                             |
+| `Task`        | `spec.sidecars[].volumeMounts.mountPath`                        |
+| `Task`        | `spec.sidecars[].volumeMounts.subPath`                          |
 | `Task`        | `spec.sidecars[].command`                                       |
 | `Task`        | `spec.sidecars[].args`                                          |
 | `Task`        | `spec.sidecars[].script`                                        |
 | `Task`        | `spec.workspaces[].mountPath`                                   |
 | `TaskRun`     | `spec.workspaces[].persistentVolumeClaim.claimName`             |
-| `TaskRun`     | `spec.workspaces[].configmap.name`                              |
+| `TaskRun`     | `spec.workspaces[].configMap.name`                              |
 | `TaskRun`     | `spec.workspaces[].secret.secretName`                           |
-| `TaskRun`     | `spec.workspaces[].projected.sources[].configMap.name`          |
 | `TaskRun`     | `spec.workspaces[].projected.sources[].secret.name`             |
+| `TaskRun`     | `spec.workspaces[].projected.sources[].configMap.name`          |
```

Comment:
This order is consistent with the declaration order of fields in the Go struct, making it easy to see if any fields are missing.

https://github.com/tektoncd/pipeline/blob/83c2f5b633e434eec9d8c7ee9bd5bff30e9f70fc/vendor/k8s.io/api/core/v1/types.go#L1721-L1733